### PR TITLE
#867: Remove ksonnet page and add kustomize page

### PR DESCRIPTION
**Pull request by scottilee**
* Removed the ksonnet page (technically, I `git mv`ed it to kustomize.md page)
* Added content about what kustomize is and how to install it

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/913)

### CODE REVIEW

File: content/docs/components/kustomize.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      export PATH=$PATH:${HOME}/bin/kustomize
+      ```
+
+## More about kustomize
+
+kustomize lets you customize raw, template-free YAML files for multiple
+purposes, leaving the original YAML untouched and usable as is.
+
+kustomize directories can be built using `kustomize build <kustomization_directory>`
+and applied using `kustomize build <kustomization_directory> | kubectl apply -f -`.
```
Comment (kbhawkey) [inline content/docs/components/kustomize.md]: - Do you want to mention the role of kustomization.yaml in more detail, or how Kubeflow has developed manifests to configure and deploy Kubeflow components?
- Is kustomize directly manipulated by the Kubeflow user, or are the commands embedded in scripts/templates?
- The directories contain manifest files (kustomization.yaml); the command takes the top level directory in the configuration file hierarchy. Does the user directly edit the manifest files to change the way Kubeflow is deployed?
- Do you want to provide a link to the Kubeflow manifest examples repo?

Comment (richardsliu) [reply content/docs/components/kustomize.md]: +1, please include a link to https://github.com/kubeflow/manifests/blob/master/README.md

Comment (scottilee) [reply content/docs/components/kustomize.md]: I tried adding some of this detail but I don't have any hands on experience so it was difficult to add all the information. I'd appreciate any help!

Comment (richardsliu) [reply content/docs/components/kustomize.md]: I think it is ok to leave out some of these details. The kfctl tool acts as a wrapper around the kustomize build commands, so we don't need to explain how kustomization.yaml is organized.

The user only needs to change manifest files if further customization is needed, in which case this page should help: https://www.kubeflow.org/docs/gke/customizing-gke/

Comment (scottilee) [reply content/docs/components/kustomize.md]: Thanks @richardsliu I added that link.

**Pull request merged**